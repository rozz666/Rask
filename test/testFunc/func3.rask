f() -> void
{
    print(1);
}

g() -> void
{
    print(2);
}
