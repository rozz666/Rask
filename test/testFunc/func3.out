func3.rask: error: missing main function
